V2/gameFuseJsonHelper.js

```diff
@@ -13,10 +13,6 @@ class GameFuseJsonHelper {
             return GameFuseUser.UserCache[userData.id]
         }
 
-        const attributes = userData.game_user_attributes && this.formatUserAttributes(userData.game_user_attributes);
-        const purchasedStoreItems = userData.game_user_store_items && userData.game_user_attributes.map(item => this.convertJsonToStoreItem(item));
-        const leaderboardEntries = userData.leaderboard_entries && userData.leaderboard_entries.map(entryData => this.convertJsonToLeaderboardEntry(entryData));
```

Comment:
I realized that these never get set through this method (`convertJsonToUser`), they only get set in subsequent requests. See the method: `setFullUserData` further below.

Reply:
Do they also come through setFullUser Data? Is this redundant?

Reply:
They do not come through `setFullUserData`. They get set in `downloadStoreItems()` and `downloadAttributes()`. So this code was setting them only if they come through, but there was no time when they come through here.